cli/src/action/user/mod.rs

```diff
@@ -52,22 +52,46 @@ fn display_splinter_users(
         .with_auth(create_cylinder_jwt_auth(signer.clone())?)
         .build()?;
 
-    let biome_users = client
-        .list_biome_users()?
-        .into_iter()
-        .map(ClientSplinterUser::from);
-
-    let biome_oauth_users = client
-        .list_oauth_users()?
-        .data
-        .into_iter()
-        .map(ClientSplinterUser::from);
+    let biome_users = match client.list_biome_users() {
+        Ok(users) => Some(users.into_iter().map(ClientSplinterUser::from)),
+        Err(e) => {
+            info!(
+                "Ignore if biome credentials are not enabled: Biome users not listed: {}",
```

Comment:
This error message will be confusing if the CLI is just pointed at the wrong HTTP port, as the user is unlikely to understand what we mean by "enabled" here. Suggest simplifying this to Just The Facts: "Unable to retrieve Biome users: {}".

If both of these fail, should that still be an error?

Reply:
Updated both log messages. If both fail the match statement on line 76
`let users = match (biome_users, biome_oauth_users) {...}`  will catch it and return an error.